### DISCUSSION TITLE
Use percentages for an axis?

### DISCUSSION THREAD

**Discussion by BenWoodworth** [JetBrains/lets-plot-kotlin#54] [Q&A] [answered]
I want to make the y-axis use percentages instead of decimals (.5% instead of .005). The ggplot solution seems to be `+ scale_y_continuous(labels = scales::percent)`, but I can't figure out the equivalent with the Kotlin API. Is this possible/easy to do?

**Comment by alshan** [comment]
In the next release we are adding the "format" option to scales.
For your example you will configure the scale as:  
`scale_y_continuous(format=".1%")`

The next release is coming soon - right after St.Patrick's I hope.

In the current version you can only format tooltips.

**Comment by alshan** [comment, marked as answer]
Hi @BenWoodworth , since [v1.3.0](https://github.com/JetBrains/lets-plot-kotlin/releases/tag/v1.3.0) the formatting option is available on positional scales. Now you can write `scale_y_continuous(format=".1%")` to format 0.005 as .5%  .

**Reply by darylz** [reply]
Thanks! But in this way, the tooltip format has not been changed. It's good to update the default tooltip format when scale_x/y is set?

**Reply by alshan** [reply]
Hi, right, formatting in tooltip is configured separately: https://github.com/JetBrains/lets-plot-kotlin/blob/master/docs/tooltips.md

However, it seems to make sense to inherit axis formatting in tooltips, thanks.

**Reply by alshan** [reply]
I've submitted a feature request: https://github.com/JetBrains/lets-plot-kotlin/issues/229

**Comment by BenWoodworth** [comment]
Thanks you! :)
